containers/containers.go

```diff
@@ -27,7 +29,10 @@ type RuntimeInfo struct {
 
 type Store interface {
 	Get(ctx context.Context, id string) (Container, error)
-	List(ctx context.Context, filter string) ([]Container, error)
+
+	// List returns containers that match one or more of the provided filters.
```

Comment:
No worries. I went back and forth on this a bunch. If you want to sit down and go over it and make sure the model is solid, let's do so.